bf_code.b

```diff
@@ -0,0 +1,177 @@
+[-]
```

Comment:
what meaning this line?
do nothing

Reply:
nothing.[-] its for reseting current cell to zero . its for making sure that current cell be zero. 
i can change it